packages/fast-form-generator-react/src/form/form-item.align-horizontal.tsx

```diff
@@ -1,16 +1,25 @@
 import * as React from "react";
 import { IFormItemComponentMappingToProperyNamesProps } from "./form-item";
+import styles from "./form-item.align-horizontal.style";
+import { IFormItemAlignHorizontalClassNameContract } from "../class-name-contracts/";
+import manageJss, { IJSSManagerProps } from "@microsoft/fast-jss-manager-react";
+import { IManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
 
 /**
  * Schema form component definition
  * @extends React.Component
  */
-class FormItemAlignHorizontal extends React.Component<IFormItemComponentMappingToProperyNamesProps, {}> {
-
+/* tslint:disable-next-line */
+class FormItemAlignHorizontal extends React.Component<IFormItemComponentMappingToProperyNamesProps & IManagedClasses<IFormItemAlignHorizontalClassNameContract>, {}> {
     public render(): JSX.Element {
         return (
-            <div>
-                <label htmlFor="alignHorizontal">{this.props.label}</label>
+            <div className={this.props.managedClasses.formItemAlignHorizontal}>
+                <label
+                    className={this.props.managedClasses.formItemAlignHorizontal_label}
+                    htmlFor="alignHorizontal"
```

Comment:
can you change the alignHorizontal to be this.props.dataLocation and use that for the input?

Reply:
Having trouble understanding what you are asking for based off of how git does commenting...